modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -14,11 +14,14 @@
 """The module defines base interface for a partition of a Modin DataFrame."""
 
 from abc import ABC
-from modin.pandas.indexing import compute_sliced_len
 from copy import copy
 
+import pandas
 from pandas.api.types import is_scalar
 
+from modin.pandas.indexing import compute_sliced_len
+from modin.core.storage_formats.pandas.utils import length_fn_pandas, width_fn_pandas
+
 
 class PandasDataframePartition(ABC):  # pragma: no cover
```

Comment:
@anmyachev, we should create `BaseDataframePartition` class, which will define an interface to be implemented by child classes. Thus, then correct docstrings and docs for this class. Would you be able to do that?

Reply:
yes, to be more precise, `modin/core/dataframe/base/partitioning/partition.py`.

Reply:
https://github.com/modin-project/modin/issues/4194